raw/src/types/text.rs

```diff
@@ -0,0 +1,36 @@
+use std::path::Path;
+
+use bytes::Bytes;
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Ord, Eq, Hash)]
+pub struct Text(pub(crate) Bytes);
+
+impl Text {
+    pub fn as_str(&self) -> &str {
+        unsafe { std::str::from_utf8_unchecked(self.0.as_ref()) }
```

Comment:
I'll take a look at this later when I have more time, but I'm hesitant to let this pass, unless we make the `Bytes` in `Text` private, just to be sure that `Text` can only be safely constructed, guaranteeing that this method is always safe

Reply:
Just changed to private, It turned out that there is no need to leave it public.

I've added `Text` type because I think we need in efficient way to both clone and pass unicode data (file name and file path for now) through multithread tokio runtime.

There is an option to remove `Text`, use `Bytes` and perform a runtime check in `telegram-bot` with `std::str::from_utf8`, and add another error variant.

Reply:
Or we can just use `Arc<String>` for file name and `Arc<PathBuf>` for path as well. Let me think about it.